src/compiler/README.md

```diff
@@ -1,17 +1,27 @@
 # Compiler
-This section documents the Rust compiler itself, its APIs, and how to
-contribute and provide bug fixes for the compiler.
+Rust's compiler team are responsible for maintaining the Rust compiler, improving its performance
+and considering the stabilisation of compiler features.
```

Comment:
I see a new file `proposals-and-stabilizations.md`.

Which spelling of `stabili{z,s}ation` do we prefer? :-)